process/templates/PROJECT_NAME_sandbox_stage.md

```diff
@@ -0,0 +1,13 @@
+## Application for creating a new project at sandbox stage
+
+### List of project maintainers
+The project must have a minimum of two maintainers with different company affiliations.
+  * "name, affiliation, GitHub ID"
+
+### Mission of the project
+The project must be aligned with the OpenSSF mission and either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
+  * "description of the project mission"
+
+### IP policy and licensing due dilligence
+If contributing an existing Project to the OpenSSF, the contribution must undergo license and IP due diligence by the Linux Foundation (LF).
+  * "yes / no / not applicable. If yes, provide a link to the corresponding GitHub issue."
```

Comment:
```suggestion
  * "yes / no / not applicable. If yes, provide a link to the corresponding GitHub issue."
  
  ### Project References
  The project should provide a list of existing resources with links to the repo, website, a roadmap, contributing guide, demos and walkthroughs, and other material to showcase the existing breadth, maturity, and direction of the project.
 | Reference | URL |
|---|---|
| Repo |   |
| Website |   | 
| Contributing |   |
| Roadmap |   | 
| Demos |    |
| Other |    |

Reply:
Given the discussion in #105, i.e., project can directly join at the incubation stage, I suggest to add the requirement for an IP policy review and the table of references to the incubation template file as well. Objections?